src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -30,23 +32,33 @@ public abstract class PredictionTransformerBase<TModel> : IPredictionTransformer
         protected readonly ISchemaBindableMapper BindableMapper;
         protected readonly ISchema TrainSchema;
 
-        public string FeatureColumn { get; }
+        public string[] FeatureColumn { get; }
```

Comment:
>FeatureColumn [](start = 24, length = 13)

oh no... I don' like this change already.

Forcing ALL predictors to expose parallel arrays of feature columns is not a great change

Reply:
I don't think there's much overhead from it. Do you think it will cause problems?